Examples/BackgroundTasks/Sources/main.swift

```diff
@@ -53,5 +53,5 @@ struct BackgroundProcessingHandler: LambdaWithBackgroundProcessingHandler {
 }
 
 let adapter = LambdaCodableAdapter(handler: BackgroundProcessingHandler())
-let runtime = LambdaRuntime.init(handler: adapter)
+let runtime = try LambdaRuntime.init(handler: adapter)
```

Comment:
I think creating multiple handlers should be fine. We should throw in the `run` method, if another lambda has already started.

Reply:
Good idea, l see the benefit of this approach. Let me work on this